src/components/tutorial-meta/components/interactive-lab-button/interactive-lab-button.module.css

```diff
@@ -0,0 +1,30 @@
+.labButton {
```

Comment:
This is rendering larger than expected, I think because of the border, but I wouldn't really worry about it for this PR because I just started the `Button` component and it should be ready for use soon. 😊

Reply:
Sounds good! I'll double check the spec again also.